src/Codec/Xlsx/Parser/Stream.hs

```diff
@@ -565,6 +571,26 @@ findName :: ByteString -> SheetValues -> Maybe SheetValue
 findName name = find ((name ==) . fst)
 {-# INLINE findName #-}
 
+setStyle ::
+  ( MonadError SheetErrors m
+     , HasSheetState m
+ )
+  => SheetValues -> m ()
+setStyle list = do
```

Comment:
Can we tidy up the formatting here? The signature is fewer than 80 characters, so I would write this as

```suggestion
setStyle :: (MonadError SheetErrors m, HasSheetState m) => SheetValues -> m ()
setStyle list = do
```